lib/galaxy/model/__init__.py

```diff
@@ -1141,6 +1175,11 @@ def attempt_create_private_role(self):
         with transaction(session):
             session.commit()
 
+    def dictify_objectstore_usage(self) -> List[UserObjectstoreUsage]:
+        session = object_session(self)
+        rows = calculate_disk_usage_per_objectstore(session, self.id)
+        return [UserObjectstoreUsage(object_store_id=r[1], total_disk_usage=r[0]) for r in rows if r[1]]
```

Comment:
```suggestion
        return [UserObjectstoreUsage(object_store_id=r.object_store_id, total_disk_usage=r.coalesce) for r in rows if r.object_store_id]
```
You could do this - I think this is much more readable and less error-prone. Also, given that you can you these keys, maybe use an alias for "coalesce"? i.e. `SELECT COALESCE..... AS usage` or something like that?

Reply:
Done - thanks for the heads up on how to use that!